### PR TITLE
Ba-12 Remove inventory item

### PR DESCRIPTION
**Pull request by lyio**
Implements deletion of single elements

### CODE REVIEW

**Comment by lyio** [comment]
@flashback2k14 @hetec for review

**Comment by flashback2k14** [comment]
maybe add a confirm dialog before delete!?

File: socobo-inventory.html [hunk 1]
```diff
@@ -18,11 +18,22 @@
...
       <socobo-inventory-create id="inventoryCreateDialog"></socobo-inventory-create>
+      
+      <paper-dialog id="confirmationDialog" modal>
+        <h2>Remove item</h2>
+        <span>Are you sure you want to remove this item?</span>
+        <div class="buttons">
+          <paper-button dialog-dismiss>Cancel</paper-button>
+          <paper-button dialog-confirm on-click="_confirmDelete">Ok</paper-button>
+        </div>
+      </paper-dialog>
```
Comment (lyio) [inline socobo-inventory.html:32]: This is the confirmation dialog. I am unsure whether this limited functionality (essentially just using the `paper-dialog`) warrants a whole Polymer element

**Comment by flashback2k14** [comment]
i.O.

**Pull request merged**